.rat-excludes

```diff
@@ -35,6 +35,7 @@ dagre-d3.min.js
 graphlib-dot.min.js
 sorttable.js
 vis.min.js
+vis.js
```

Comment:
In that case I would prefer to leave it out if possible. This is bringing in a 30k lines file into our repo, not to mention that it slows down the page load speed.